.github/workflows/ci.yml

```diff
@@ -16,8 +16,12 @@ jobs:
         run: |
           # We are not yet using the new naming in the deploy workflow
           docker build -t gitbot:latest .
-          docker run -p 8080:8080 -d -e GOOGLE_APPLICATION_CREDENTIALS -e FAST_STARTUP=1 gitbot:latest
-          sleep 10 && curl --request POST http://0.0.0.0:8080
+
+      - name: Run image & ingest
+        run: |
+          container_id=$(docker run -p 8080:8080 -d -e DRY_RUN=1 -e FAST_STARTUP=1 gitbot:latest)
```

Comment:
This allows making the output of the container show up in the logs since we can inquire the daemonized container run.